release/release_tests.yaml

```diff
@@ -4791,14 +4791,9 @@
   group: core-daily-test
   working_dir: nightly_tests
 
-  python: "3.8"
   frequency: nightly
   team: core
   cluster:
-    byod:
-      runtime_env:
-        - RAY_MAX_LIMIT_FROM_API_SERVER=1000000000
```

Comment:
is this needed?

Reply:
The env is provided by the test owner https://github.com/ray-project/ray/blob/master/release/nightly_tests/stress_tests/state_api_app_config.yaml#L3 so I assume it is needed yeah, but I think byod has issues setting runtime env